src/app/components/common/event-preview/event-preview.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="event">
+    <img src="{{ event.image }}" alt="{{ event.title }}" class="w-100 rounded-lg">
```

Comment:
use safe navigation operator (?) when you are binding the data! {{event?.iamge}}

Reply:
Done